api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirTaskDao.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.dao;
+
+import javax.validation.constraints.NotNull;
+
+import org.openmrs.api.db.DAOException;
+import org.openmrs.module.fhir2.Task;
+
+public interface FhirTaskDao {
+
+	public Task saveTask(Task task) throws DAOException;
```

Comment:
I would prefer if we didn't explicitly document or throw a DAOException here. We want to keep the reliance on core as minimal as possible.